src/components/chips/js/chipsDirective.js

```diff
@@ -65,6 +65,8 @@
    * @param {string=} placeholder Placeholder text that will be forwarded to the input.
    * @param {string=} secondary-placeholder Placeholder text that will be forwarded to the input,
    *    displayed when there is at least on item in the list
+   * @param {boolean=} md-removeable Allows only to remove a chip from the list through the delete buttons
```

Comment:
I think I would change this to the following:

> Enables or disables the deletion of chips through the removal icon or the Delete/Backspace key. Defaults to true.

Right now, it sounds like it somehow changes what controls are used for deletion; not the deletion capability itself :smile: